src/lib/components/Carousel.svelte

```diff
@@ -39,41 +45,58 @@
     const totalItems = Math.round(scrollWidth / offsetWidth);
     const activeIndex = Math.round(scrollLeft / scrollWidth * totalItems);
 
+    // Vind de indicator voor de actieve items
     const indicator = document.querySelector('.carousel-indicator-span');
+    // Vind alle individuele indicatoren
     const indicators = indicator.querySelectorAll('.carousel-indicator-span-span');
+    // Verwijder de klasse 'is-active' van alle indicatoren
     indicators.forEach((indicator, index) => {
       indicator.classList.remove('is-active');
     });
+    // Voeg de klasse 'is-active' toe aan de indicator van het actieve item
     indicators[activeIndex].classList.add('is-active');
   }
 
+  // Functie die wordt uitgevoerd wanneer het component wordt gemonteerd
   onMount(() => {
+    // Zorg ervoor dat de knoppen en indicatoren zichtbaar zijn bij als de browser JS heeft ingeschakeld
     const carouselElements = document.querySelectorAll('.carousel-link, .carousel-indicator');
     carouselElements.forEach(function(element) {
       element.hidden = false;
     });
   });
+
+  // Zorg ervoor dat de carousel informatie button niet zichtbaar is waneer JS ingeschakeld is
+  onMount(() => {
+    const carouselElements = document.querySelectorAll('.carousel-info-button');
+    carouselElements.forEach(function(element) {
+      element.style.display = 'none';
+    });
+  });
 </script>
 
 
+
 <h2>Wensen</h2>
 
 {#if data}
 <section class="carousel-container">
-  <!-- Previous button -->
+  <!-- Vorige knop -->
   <a href="#" class="carousel-link left-arrow" on:click={scrollLeftOrRight} title="Volgende slide" hidden>
     <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="carousel-arrow">
       <polyline points="15 18 9 12 15 6"></polyline>
     </svg>
   </a>
 
+  <!-- Carousel container -->
   <div class="carousel">
     <div class="carousel-inner">
       {#each data.wishes as wish}
       <div class="carousel-item">
-        <img class="carousel-image" src={wish.image.url} alt="" decoding="async" width="150px" height="150px" loading="lazy"/>
+        <img class="carousel-image" src={wish.image.url} alt="" decoding="async" width="150" height="150" loading="lazy"/>
```

Comment:
Vul je alt attribuut nog in, deze is momenteel leeg.

Reply:
De alt is expres leeg gelaten omdat de afbeelding decoratief is en niets toevoegt aan de inhoud voor de schermlezer.

<img width="620" alt="Schermafbeelding 2024-03-27 om 18 22 50" src="https://github.com/fdnd-agency/crossmarx/assets/112856019/25d53bd9-e04a-4ad7-90f1-198057cc30c3">